manifests/config.pp

```diff
@@ -185,10 +186,12 @@
         include 'systemd'
         systemd::unit_file {'prometheus.service':
           content => template('prometheus/prometheus.systemd.erb'),
+          notify  => [Class['systemd::systemctl::daemon_reload'], Class['prometheus::run_service']],
```

Comment:
I think `systemd::unit_file` handles `Class['systemd::systemctl::daemon_reload']`, so that should not be required here.